exercises/luhn/example.cpp

```diff
@@ -0,0 +1,44 @@
+#include "luhn.h"
+
+using namespace std;
+
+namespace luhn{
+
+    bool valid(std::string const& input_str)
```

Comment:
Please run `clang-format --style="{BasedOnStyle: google, IndentWidth: 4}" -i example.cpp example.h` to fix the formatting.

Clang-format is installed with clang (or possibly llvm). If you don't have access to that I can do it myself after this is finished.

Reply:
Can we make travis fail if the formatting is wrong?

Reply:
Yes we can. Getting us to standardize on formatting and running it on a git hook and in travis has been on my backlog, but it's lower pri than the boost stuff.